README.md

```diff
@@ -264,11 +264,11 @@ See examples in [sample-mod/fabric.mod.json](https://github.com/FabricMC/fabric-
 ## Bundled libraries
 
 ```
-org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72
-org.jetbrains.kotlin:kotlin-reflect:1.3.72
-org.jetbrains:annotations:17.0.0
-org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.7
-org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:1.3.7
+1.4.0
+1.4.0
+17.0.0
+1.3.9
+1.3.9
 ```
```

Comment:
This... doesn't seem intentional?

Reply:
yu are correct.. i lost stuff there, oops